### PR TITLE
adding binary that automatically sets the full icu

### PR DESCRIPTION
**Pull request by martinheidegger**
In a CI process it can be rather tricky to get node to use full icu. This PR adds a binary that starts directly with the icu data file if necessary which should make many processes a lot simpler.

### CODE REVIEW

**Comment by martinheidegger** [comment]
I used it here: https://github.com/maxogden/extract-zip/pull/116/files#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519R17

**Comment by srl295** [comment]
I like this idea!

**Comment by srl295** [comment]
@martinheidegger can you give some examples of use, perhaps in the README?

File: node-full-icu.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env node
+const { spawn } = require('child_process')
+const data = require('./full-icu')
+const env = data.icu_small ? {
+  ...process.env,
+  NODE_ICU_DATA: data.datPath()
+} : process.env
+
+spawn('/usr/bin/env', ['node', ...process.argv.slice(2)], { env, stdio: 'inherit' })
```
Comment (srl295) [inline node-full-icu.js:9]: /usr/bin/env would only work on *nix?

Comment (srl295) [reply node-full-icu.js:9]: i think things like npm-cli sublaunch node, must be a portable way to do it

**Comment by martinheidegger** [comment]
There are a bunch of things that I should better be testing, so in #59 I started the first step to get a CI test system up and running, so these can also be tested (since github action can be run in windows as well)

**Review by srl295 — APPROVED** [review]
going to approve for now

**Pull request merged**